fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheAnalyzer.java

```diff
@@ -595,6 +607,14 @@ private CacheTable getSelectedPartitionLastUpdateTime(OlapScanNode node) {
         return cacheTable;
     }
 
+    private CacheTable buildCacheTableForHiveScanNode(HiveScanNode node) {
+        CacheTable cacheTable = new CacheTable();
+        cacheTable.table = node.getTargetTable();
+        cacheTable.partitionNum = node.getReadPartitionNum();
+        cacheTable.latestTime = cacheTable.table.getLastUpdateTime();
```

Comment:
This is a mocked `latestTime`?

Reply:
Noop, the lastUpdateTime of hive table will be changed at some situations, you can see here: #21873